pages/checkout/index.js

```diff
@@ -0,0 +1,158 @@
+import React, { useContext, useState } from 'react';
+
+import { CartContext } from '../../src/contexts/CartContext';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import { Link } from 'next/link';
+import Button from '@material-ui/core/Button';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+import InputLabel from '@material-ui/core/InputLabel';
+import { getMenu } from '../../lib/api';
+import { useRouter } from 'next/router';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    width: '25ch',
```

Comment:
Wtf is a `ch`? 😅

Reply:
ch | Relative to the width of the "0" (zero)
-- | --

Reply:
Aight, if that works for you :D